docs/sources/upgrade-guide/when-to-upgrade/index.md

```diff
@@ -96,49 +96,63 @@ For self-managed Grafana (both Enterprise and OSS), the support for versions fol
-| 11.2.x                    | August 27, 2024    | May 27, 2025         | Not Supported                 |
-| 11.3.x                    | October 22, 2024   | July 22, 2025        | Not Supported                 |
-| 11.4.x                    | December 5, 2024   | September 5, 2025    | Security & Critical Bugs Only |
-| 11.5.x                    | January 28, 2025   | October 28, 2025     | Security & Critical Bugs Only |
-| 11.6.x (Last minor of 11) | March 25, 2025     | June 25, 2026        | Security & Critical Bugs Only |
-| 12.0.x                    | May 5, 2025        | February 5, 2026     | Security & Critical Bugs Only |
-| 12.1.x                    | July 22, 2025      | April 22, 2026       | Full Support until next minor |
-| 12.2.x                    | September 23, 2025 | June 23, 2026        | Yet to be released            |
-| 12.3.x                    | November 18, 2025  | August 18, 2026      | Yet to be released            |
+| **Version**               | **Release date**   | **Support end date** | **Support level**  |
+| ------------------------- | ------------------ | -------------------- | ------------------ |
+| 10.2.x                    | October 24, 2023   | July 24, 2024        | Not Supported      |
+| 10.3.x                    | January 23, 2024   | October 23, 2024     | Not Supported      |
+| 10.4.x (Last minor of 10) | March 5, 2024      | June 5, 2025         | Not Supported      |
+| 11.0.x                    | May 14, 2024       | February 14, 2025    | Not Supported      |
+| 11.1.x                    | June 25, 2024      | April 23, 2025       | Not Supported      |
+| 11.2.x                    | August 27, 2024    | May 27, 2025         | Not Supported      |
+| 11.3.x                    | October 22, 2024   | July 22, 2025        | Not Supported      |
+| 11.4.x                    | December 5, 2024   | September 5, 2025    | Patch Support      |
+| 11.5.x                    | January 28, 2025   | October 28, 2025     | Patch Support      |
+| 11.6.x (Last minor of 11) | March 25, 2025     | June 25, 2026        | Patch Support      |
+| 12.0.x                    | May 5, 2025        | February 5, 2026     | Patch Support      |
+| 12.1.x                    | July 22, 2025      | April 22, 2026       | Patch Support      |
+| 12.2.x                    | September 23, 2025 | June 23, 2026        | Yet to be released |
+| 12.3.x                    | November 18, 2025  | August 18, 2026      | Yet to be released |
 
 ## How are these versions supported?
 
-The level of support changes as new versions of Grafana are released. Here are the key details:
+Self-managed Grafana follows semantic-like versioning (MAJOR.MINOR.PATCH). Here's how different types of releases work:
 
-- **Full Support**:
+- **Major releases** (e.g., 12.5.8, 13.0.0):
+  - Awesome new features as well as significant architectural improvements and modernizations
+  - May include breaking changes that require migration steps
```

Comment:
It's a translation thing. Might is clearer than may when translating from English
```suggestion
  - Might include breaking changes that require migration steps
```